tests/pom.xml

```diff
@@ -232,7 +232,7 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
                     <env>
                       <LOGGING_CONFIG>file:///etc/hono/logback-spring.xml</LOGGING_CONFIG>
                       <SPRING_CONFIG_LOCATION>file:///etc/hono/</SPRING_CONFIG_LOCATION>
-                      <SPRING_PROFILES_ACTIVE>prod</SPRING_PROFILES_ACTIVE>
+                      <SPRING_PROFILES_ACTIVE>dev</SPRING_PROFILES_ACTIVE>
```

Comment:
you may, of course, set this to dev on your local machine, but in the repo, we want to keep this set to prod.

Reply:
thanks. forgot to change it back